Launcher/QuantConnect.Lean.Launcher.csproj

```diff
@@ -45,6 +45,90 @@
     <Reference Include="DynamicInterop, Version=0.7.4.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\DynamicInterop.0.7.4\lib\net40\DynamicInterop.dll</HintPath>
     </Reference>
+    <Reference Include="IKVM.AWT.WinForms, Version=8.1.5717.0, Culture=neutral, PublicKeyToken=13235d27fcbfff58, processorArchitecture=MSIL">
```

Comment:
Interestingly, it looks like this wasn't required before. Has something changed?

Reply:
Looking at the IKVM version history on NuGet (https://www.nuget.org/packages/IKVM/), we upgraded from v7.3 to v8.1 so the new version includes 2 years of changes (as well as a few new modules). I don't exactly know which is the minimal set of DLLs required for LEAN usage and the dependency tree, but since debugging MEF loader errors is quite painful and time-consuming, I opted for the safe solution for now -- including all of the references added automatically by the NuGet package manager.